### PR TITLE
fix(condo): DOMA-6371 hide units&section selector if 0 or 2+ properti…

### PR DESCRIPTION
**Pull request by AleX83Xpert**
…es selected

### CODE REVIEW

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx [hunk 1]
```diff
@@ -852,45 +852,49 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
...
-                                                    <Form.Item
-                                                        name='unitNames'
-                                                        label={
-                                                            !isOnlyOnePropertySelected || selectedPropertiesLoading || !isEmpty(selectedSectionIds)
+                                                {isOnlyOnePropertySelected && (
+                                                    <Col span={11}>
+                                                        <Form.Item
+                                                            name='unitNames'
+                                                            label={
+                                                                !isOnlyOnePropertySelected || selectedPropertiesLoading || !isEmpty(selectedSectionIds)
```
Comment (Kekmus) [inline apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx:860]:
```suggestion
                                                                selectedPropertiesLoading || !isEmpty(selectedSectionIds)
```

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx [hunk 2]
```diff
@@ -852,45 +852,49 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
...
+                                                                onChange={handleChangeUnitNameInput}
+                                                            />
+                                                        </Form.Item>
+                                                    </Col>
+                                                )}
+                                                {isOnlyOnePropertySelected && (
+                                                    <Col span={11} offset={2}>
+                                                        <Form.Item
+                                                            name='sectionIds'
+                                                            label={!isOnlyOnePropertySelected || selectedPropertiesLoading || !isEmpty(selectedUnitNames)
```
Comment (Kekmus) [inline apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx:882]:
```suggestion
                                                            label={selectedPropertiesLoading || !isEmpty(selectedUnitNames)
```

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx [hunk 3]
```diff
@@ -852,45 +852,49 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
...
-                                                            property={selectedProperties[0]}
-                                                            onChange={handleChangeSectionNameInput(selectedProperties[0])}
-                                                            mode='multiple'
-                                                        />
-                                                    </Form.Item>
-                                                </Col>
+                                                                : SectionsLabel}
+                                                        >
+                                                            <SectionNameInput
+                                                                disabled={!isOnlyOnePropertySelected || selectedPropertiesLoading || !isEmpty(selectedUnitNames)}
```
Comment (Kekmus) [inline apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx:890]:
```suggestion
                                                                disabled={selectedPropertiesLoading || !isEmpty(selectedUnitNames)}
```

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx]: done

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx]: done

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx (hunk 3 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx]: done

**Review by Kekmus — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx [hunk 4]
```diff
@@ -852,45 +852,49 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
...
+                                                                multiple={true}
+                                                                property={selectedProperties[0]}
+                                                                allowClear={false}
+                                                                loading={selectedPropertiesLoading || !isEmpty(selectedSectionIds)}
+                                                                onChange={handleChangeUnitNameInput}
+                                                            />
+                                                        </Form.Item>
+                                                    </Col>
+                                                )}
+                                                {isOnlyOnePropertySelected && (
```
Comment (pahaz) [inline apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx:878]: it's better to use one `isOnlyOnePropertySelected` with one jsx group `<><>`

**Pull request merged**